backend/result/models.py

```diff
@@ -1,18 +1,38 @@
-import json
-
 from django.db import models
 from django.utils import timezone
 
 from django.core.exceptions import ValidationError
 
 
-# Create your models here.
 class Result(models.Model):
-    """Score for each step in a session"""
-    session = models.ForeignKey('session.Session', on_delete=models.CASCADE, blank=True, null=True)
-    participant = models.ForeignKey('participant.Participant', on_delete=models.CASCADE, blank=True, null=True)
+    """A model to register participant responses to the database.
+
+    Note that a Result can be either tied to a Session, which is the common way to log responses to tasks during an experiment,
+    or to a Participant, which can be a consent form or a demographic question.
```

Comment:
mkdocs does have a

```mkdocs
Note:
    Lorem ipsum
```

syntax. Would it make sense to use that here too or would you prefer it to be shown on top?